docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -321,7 +321,7 @@ However, if your model contains more than one input, Model Optimizer is able to
 
 #### Q49. What does the message "Data type is unsupported" mean? <a name="question-49"></a>
 
-**A** : Model Optimizer cannot convert the model to the specified data type. Currently, FP16 and FP32 are supported. Make sure you specify the data type with the `--data_type` flag. The available values are: FP16, FP32, half, float.
+**A** : Model Optimizer cannot convert the model to the specified data type. Currently, the following types are supported: bool, float16, float32, double, int8, int16, int32, int64, uint8, uint16, uint32, uint64, str.
```

Comment:
What does it mean: `convert model to data type`? How can we convert model to a data type?

Reply:
I mean, in the context of fp16/fp32 it is _probably_ possible to say so, but by listing all the data types you extended the context, and now it doesn't have the original meaning, for example how can you convert model to `bool`?

Reply:
Replaced with "Model Optimizer cannot read the value of the specified data type."